tests/hpc/before_test.pm

```diff
@@ -16,6 +16,14 @@ use warnings;
 use testapi;
 use utils;
 use lockapi;
+use registration 'add_suseconnect_product';
+
+sub ensure_registered {
+    assert_script_run('SUSEConnect --cleanup', 200);
+    add_suseconnect_product('SLES', get_var('VERSION'), get_var('ARCH'), ' -e testing@suse.com -r ' . get_required_var('SCC_REGCODE'));
+    add_suseconnect_product('sle-module-hpc',           get_var('VERSION'));
+    add_suseconnect_product('sle-module-web-scripting', get_var('VERSION'));
```

Comment:
I guess you just use `get_var` because you know that we commonly have ARCH as well as VERSION, but why not just use `get_required_var` for the sake of being explicit?